src/command_prechecks.ts

```diff
@@ -47,7 +47,7 @@ export default class CommandPrechecks {
             messageContext.guildID
         );
 
-        if (session.isListeningSession()) {
+        if (session.isListeningSession() && interaction) {
```

Comment:
What about an in session check triggered by a non-interaction command, with the bot and user in different channels? We'd skip entering here and would be casting the ListeningSession as a GameSession

Reply:
This conditional is only for interaction response via `tryCreateInteractionErrorAcknowledgement`. If we want to make it the same behavior for non-interactions then we need another PR for it.